src/js/fc.js

```diff
@@ -69,6 +69,7 @@ const FC = {
     VTXTABLE_BAND: null,
     VTXTABLE_POWERLEVEL: null,
     VTX_CONFIG: null,
+    MOTOR_OUTPUT_ORDER: null,
```

Comment:
One little thing about this... I have ordered all the MSP commands by alphabetical order. Is an easy way to find what and what not exists. It will be great if you can maintain this order.